### PR TITLE
Remove undocumented and untested script

### PR DESCRIPTION
**Pull request by hmusum**
Script has an error (wrong Java class name) and gives no useful output (covered by metrics and other tools), removing

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/SearchGroupsImpl.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class SearchGroupsImpl implements SearchGroups {
     private final Map<Integer, Group> groups;
     private final double minActiveDocsPercentage;
 
-    public SearchGroupsImpl(Map<Integer, Group> groups, double minActiveDocsPercentage) {
+    public SearchGroupsImpl(Map<Integer, Group> groups  , double minActiveDocsPercentage) {
```
Comment (vekterli) [inline container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/SearchGroupsImpl.java:24]: Unintended change to spacing?

**Review by vekterli — DISMISSED** [review]
:+1:

File: container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/SearchGroupsImpl.java (hunk 1 above)
Comment (hmusum) [reply container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/SearchGroupsImpl.java]: Yes, thanks, fixed.

**Comment by hmusum** [comment]
@vekterli please take a look again

**Review by vekterli — APPROVED** [review]

**Pull request merged**